app/src/main/java/org/hackillinois/android/view/LoginActivity.kt

```diff
@@ -75,25 +75,24 @@ class LoginActivity : AppCompatActivity() {
                     }
 
                     if (getOAuthProvider() == "google") {
-                        verifyStaffRole(api, it)
+                        verifyRole(api, it, "Staff")
                     } else {
-                        JWTUtilities.writeJWT(applicationContext, it)
-                        launchMainActivity()
+                        verifyRole(api, it, "Attendee")
                     }
                 }
             }
         })
     }
 
-    private fun verifyStaffRole(api: API, jwt: String) {
+    private fun verifyRole(api: API, jwt: String, roleString: String) {
```

Comment:
I'm not a fan of putting types in variable names. You could probably just call this `role`